spec/onc_certification_g10_test_kit/export_kick_off_performer_spec.rb

```diff
@@ -12,12 +12,12 @@ def http_clients
 RSpec.describe ONCCertificationG10TestKit::ExportKickOffPerformer do
   let(:token) { 'some_token' }
   let(:group_id) { 'some_group_id' }
-  let(:param) { '_outputFormat=ndjson' }
   let(:bulk_server_url) { 'https://www.example.com' }
   let(:polling_url) { 'https://www.some_polling_url.com' }
   let(:performer) { ExportKickOffPerformerTesterClass.new }
   let(:request) { Inferno::Entities::Request.new({ headers: [header] }) }
   let(:bulk_export_url) { "#{bulk_server_url}/Group/#{group_id}/$export" }
+  let(:params) { { _outputFormat: 'appplication/fhir+ndjson', _sort: 'sample+value' } }
```

Comment:
Probably doesn't matter technically, but you misspelled `application` here.

Reply:
:slightly_frowning_face: